readme.md

```diff
@@ -5,9 +5,11 @@ Seen these cool projects on github? Pain to download? This program is for you!
 
 Installation
 ------------
-If only there were some easy way to download this program from github...
+The easiest way to install this project on your computercraft computer is by running:
 
-Copy both files in `apis/` to `/apis/`, and put `programs/github` wherever you want it.
+    pastebin run p8PJVxC4 [<branch>|<tag>]
```

Comment:
a) it’s there for testing…
b) it’s presence hurts nothing
c) you may upgrade the project to be usable with only certain version of
CC, and that may include altering install.lua, and if I’m on an older
version of CC I’d like to be able to specify what version of
computercraft-github I’m interested in using.

On Thu, Aug 25, 2016 at 2:56 AM Eric Wieser <notifications@github.com>
wrote:

> In readme.md
> <https://github.com/eric-wieser/computercraft-github/pull/4#discussion_r76196390>
> :
>
> >
> > -Copy both files in `apis/` to `/apis/`, and put `programs/github` wherever you want it.
> > +    pastebin run p8PJVxC4 [<branch>|<tag>]
>
> I don't think this makes any sense. No one should ever need to actually
> get a different branch, because they can do that by bootstrapping from
> master
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/eric-wieser/computercraft-github/pull/4/files/6e9f6c5c534b6a6f6fff30ff8e18c376b69022d8#r76196390>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA41Fm4mMg6XnOkkYDO4yAfKP8pTJ4-5ks5qjUqYgaJpZM4JoHt0>
> .
>